core/docs/ApiChangelogs/0.1.0-0.2.0.txt

```diff
@@ -0,0 +1,1471 @@
+---------------------------------
+API Annotations
+---------------------------------
+
+[A] : Added
+[R] : Removed
+[C] : Signature changed
```

Comment:
Just "Changed" should work. The tag is present on modules as well. This is basically a tree and the node has `[C]` if any descendants of it are changed.